places-rest-apis/adobe-i-o-integration/README.md

```diff
@@ -66,23 +65,35 @@ The following video guides you through the process of generating the key pair:
 To create a Places integration:
 
 1. Go to [https://console.adobe.io/](https://console.adobe.io/) and sign in with your Adobe ID.
-2. If you have access to more than one Experience Cloud organization, select the organization from the drop-down list on the left.
-3. Click **New Integration**.
-4. Select **Access an API** and click **Continue**.
-5. Under **Experience Cloud**, select **Places** as the Adobe service to which you want to integrate and click **Continue**.
+
+2. Click on the **Create integration** in the **Quick Start** section.
+
+3. Select **Access an API** (default selection) and click **Continue**.
+
+4. If you have access to more than one Experience Cloud organization, select the desired organization from the drop-down list on the top right.
+
+5. Under **Experience Cloud**, select **Places** as the Adobe service to which you want to integrate with and click **Continue** near the bottom of the page.
+
 6. Select **New integration** and click **Continue**.
 7. On the _Create a new integration_ screen, enter a name and description.
 8. Drag and drop your `xxxx_public.crt` file, created above, to the **Public keys certificates** drop zone.
-9. At the bottom of the page, click **Create integration**.
-10. After a few seconds, in the _Integration created_ screen, verify that the following message appears:
+
+9. Select a product profile. Check with System Admin for you organization if unsure which to select.
+
+10. At the bottom of the page, click **Create integration**.
+
+11. After a few seconds, in the _Integration created_ screen, verify that the following message appears:
 
     `Your integration has been created.`
 
-11. Click **Continue to integration details**. An overview of your integration with the API key, your organization ID, the technical account ID, and other details about your integrations are displayed.
+12. Click **Continue to integration details**.  
+
+13. The integration details page appears, with the name of the integration at the top. The Overview tab is shown by default which displays the API key, your organization ID, the technical account ID, and other details about your integrations.
```

Comment:
Please restructure as follows:
"13. The integration details page appears with the name of the integration at the top. 

    The **Overview** tab is shown by default and displays the API key, your organization ID, the technical account ID, and other details about your integrations."